configure.ac

```diff
@@ -3,7 +3,7 @@
 # See https://www.gnu.org/software/autoconf/manual/autoconf.html for help on
 # the macros used in this file.
 
-AC_INIT([streamly], [0.7.2], [streamly@composewell.com], [streamly])
+AC_INIT([streamly], [0.8.0], [streamly@composewell.com], [streamly], [https://streamly.composewell.com])
```

Comment:
Version should be tracked in a lot of places. We should somehow try to commonize its source.